x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-s3-overview.json

```diff
@@ -1,756 +1,766 @@
+                        "bar_color_rules": [
+                            {
+                                "id": "f703aff0-475f-11e9-a9de-e776805ecfc9"
+                            }
+                        ],
+                        "default_index_pattern": "metricbeat-*",
+                        "default_timefield": "@timestamp",
+                        "drop_last_bucket": 0,
+                        "gauge_color_rules": [
+                            {
+                                "id": "f8388670-475f-11e9-a9de-e776805ecfc9"
+                            }
+                        ],
+                        "gauge_inner_width": 10,
+                        "gauge_style": "half",
+                        "gauge_width": 10,
+                        "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+                        "index_pattern": "metricbeat-*",
+                        "interval": "\u003e=1d",
+                        "isModelInvalid": false,
+                        "series": [
+                            {
+                                "axis_position": "right",
+                                "chart_type": "line",
+                                "color": "#68BC00",
+                                "fill": "0",
+                                "formatter": "bytes",
+                                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                                "line_width": 1,
+                                "metrics": [
+                                    {
+                                        "field": "aws.s3_daily_storage.bucket.size.bytes",
+                                        "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                                        "type": "avg"
+                                    }
+                                ],
+                                "point_size": 1,
+                                "separate_axis": 0,
+                                "split_color_mode": "gradient",
+                                "split_mode": "terms",
+                                "stacked": "none",
+                                "terms_field": "aws.s3.bucket.name",
+                                "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417",
+                                "terms_size": "20"
+                            }
+                        ],
+                        "show_grid": 1,
+                        "show_legend": 1,
+                        "time_field": "@timestamp",
+                        "time_range_mode": "entire_time_range",
+                        "tooltip_mode": "show_all",
+                        "type": "top_n"
+                    },
+                    "title": "S3 Daily Storage Bucket Size in Bytes [Metricbeat AWS]",
+                    "type": "metrics"
+                }
+            },
+            "id": "2dbb8f90-4760-11e9-8062-c98a86cb6f94",
+            "migrationVersion": {
+                "visualization": "7.10.0"
```

Comment:
Btw, there are changes to much newer versions, is this needed/desired? Users will need Kibana 7.10 to use this dashboard.

Reply:
Good point, no this is not required. I will remove it! Thank you!